### PR TITLE
aodvv2: add Router Client Set implementation

### PR DESCRIPTION
**Pull request by jeandudey**
Depends on #56, #57

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -26,10 +31,6 @@ USEMODULE += gnrc_ipv6_default
 USEMODULE += gnrc_udp
```
Comment (gustavosinbandera1) [inline Makefile:31]: this USERMODULE can be removed

Comment (jeandudey) [reply Makefile:31]: Why? It's used by `gnrc_udp_sock`

Comment (gustavosinbandera1) [reply Makefile:31]: I am not sure about it, because when I did the test without that module, the result is the same, thats mean its no necessarr

Comment (jeandudey) [reply Makefile]: Resolved in #59

**Pull request merged**